File: fieldsync-inventory-backend/src/main/java/com/example/fieldsync_inventory_api/dto/inventory/seed_condition/SeedConditionResponseDTO.java
package com.example.fieldsync_inventory_api.dto.inventory.seed_condition;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.Instant;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SeedConditionResponseDTO {
    private Integer id;
    private String name;
    private String description;
    private Instant createdAt;
    private Instant updatedAt;
    private Instant deletedAt;
}

File: fieldsync-inventory-app/app/src/main/java/com/example/fieldsync_inventory_app/data/remote/dto/seed_condition/SeedConditionResponseDto.kt
package com.example.fieldsync_inventory_app.data.remote.dto.seed_condition

// SeedConditionResponseDto for the server's response
data class SeedConditionResponseDto(
    val id: Int,
    val name: String,
    val description: String?,
    val createdAt: String,
    val updatedAt: String,
    val deletedAt: String?
)
